src/applications/simple-forms/21-0966/manifest.json

```diff
@@ -2,6 +2,6 @@
   "appName": "21-0966 Intent to file a claim",
   "entryFile": "./app-entry.jsx",
   "entryName": "21-0966-intent-to-file-a-claim",
-  "rootUrl": "/intent-to-file-a-claim-form-21-0966",
+  "rootUrl": "/supporting-forms-for-claims/intent-to-file-a-claim-form-21-0966",
```

Comment:
was this changed in content-build as well? so that it will work?

Reply:
[PR here.](https://github.com/department-of-veterans-affairs/content-build/pull/1725)